frontend/public/locales/en/renew-ita.json

```diff
@@ -48,5 +48,55 @@
     },
     "back-btn": "Back",
     "continue-btn": "Continue"
+  },
+  "update-address": {
+    "page-title": "Contact information",
```

Comment:
Since "Contact Information" is a separate page

```suggestion
    "page-title": "Mailing address",
```

Reply:
oops, forgot to push an amended commit